contrib/jemalloc/include/jemalloc/jemalloc_FreeBSD.h

```diff
@@ -73,7 +73,7 @@
 #endif
 #ifdef __mips__
 #ifdef __mips_n64
-#  define LG_VADDR		64
+#  define LG_VADDR		48
```

Comment:
At least for the "old" pmap code, which is all we've ever used, this is in fact 39, I think.  48 is probably sane and I don't think lowering it to the "real" value buys us anything?